### DISCUSSION TITLE
datetime() function is not working

### DISCUSSION THREAD

**Discussion by killJoy-03** [memgraph/memgraph#2541] [Q&A] [answered]
I am using datetime function but it says Function 'DATETIME' doesn't exist.

**Comment by antejavor** [comment]
Hi @killJoy-03, thanks for asking the question. 

Can you share the info on what version of Memgraph you are running? 
What specific function you were trying to call? 

Sharing the code snippet would be great! 🙌

**Reply by killJoy-03** [reply]
memgraph version - 2.15.1
memgraphlab version - 2.13.0
RETURN datetime("2024-04-21T14:15:00-07:00[America/Los_Angeles]")
![image](https://github.com/user-attachments/assets/1ecb5842-0179-4037-bcbe-0dac1917cabd)

**Reply by antejavor** [reply, marked as answer]
Hi @killJoy-03, the problem is that this function was added in the **2.16.1**: https://memgraph.com/docs/release-notes#memgraph-v2161---may-15-2024 It is zoned data time. 

Can you try running this on version >= 2.16.1?
